skyportal/facility_apis/observation_plan.py

```diff
@@ -366,7 +366,7 @@ def delete(request_id):
                 "default": "tiling",
             },
             "exposure_time": {"type": "string", "default": "300"},
-            "filters": {"type": "string", "default": "g,r,i"},
+            "filters": {"type": "string", "default": "ztfg,ztfr,ztfi"},
```

Comment:
Do we have to have the ZTF in the defaults? Can we easily pull this value from the config?

Reply:
great idea. fixed.